app/scripts/components/analysis/define/page-hero-actions.tsx

```diff
@@ -19,7 +18,7 @@ import { composeVisuallyDisabled } from '$utils/utils';
 const SaveButton = composeVisuallyDisabled(Button);
 
 interface PageHeroActionsProps {
-  size: string;
+  size: ButtonProps['size'];
```

Comment:
I wasn't aware of this syntax ("indexed access types"). 
I feel though that we'd be better off with a distinct ButtonSize type.

Reply:
Indeed. The ui library definitely needs a typescript improvement. For the time being this is what's possible.

@nerik Maybe we can do some UI lib cleanup during flex week. I'd appreciate your input a lot :)